### PR TITLE
feat(MakeSiderealDayStream): a new task to simulate a specific LSD

### PR DESCRIPTION
**Pull request by jrs65**

### CODE REVIEW

File: draco/synthesis/stream.py [hunk 1]
```diff
@@ -352,3 +352,73 @@ def _next_time_axis(self):
         self._cur_time += nsamp * int_time
 
         return time
+
+
+class MakeSiderealDayStream(task.SingleTask):
```
Comment (cahofer) [inline draco/synthesis/stream.py:357]: just wondering if there is a better name for this task... Maybe just SiderealDayStream or SiderealStreamLSD. But that's up to you Richard.

**Review by cahofer — APPROVED** [review]
Did you test the task? I don't see anything obvious that needs change. The f-string syntax is new to me. Is that new python 3 standard?

File: draco/synthesis/stream.py (hunk 1 above)
Comment (jrs65) [reply draco/synthesis/stream.py:357]: Yeah. I couldn't think of a better one. This is vaguely in line with our conventions for other names. At somepoint we might go through all the task names and try and impose some order on them.

**Comment by jrs65** [comment]
I have indeed tested it. Thanks for the review!!

**Pull request merged**